### ISSUE TITLE
Remove `ma` bridge axioms

### ISSUE THREAD

**Issue by caufieldjh** [monarch-initiative/phenio#33]
The bridge axioms brought in for `ma` aren't contributing much and should be removed.
See `phenio-edit.owl`.

**Comment by matentzn** [comment]
I agree, ma.owl should also not be part of PHENiO, but completely covered by Uberon. Is Kevin aware that all anatomy related ingests in Monarch should be mapped to Uberon?

**Comment by caufieldjh** [comment]
@kevinschaper ☝️

See also #34

**Comment by kevinschaper** [comment]
Now I know!

I created https://github.com/monarch-initiative/monarch-ingest/issues/399

I just need to get my hands on a SSSOM file

**Comment by matentzn** [comment]
- http://purl.obolibrary.org/obo/uberon/uberon.sssom.tsv
- http://purl.obolibrary.org/obo/cl/cl.sssom.tsv

@kevinschaper can you close this issue if you feel satisfied?

**Comment by kevinschaper** [comment]
I was just re-reading https://github.com/monarch-initiative/monarch-ingest/issues/399 to catch up

Will this affect the relation graph closures that let us show phenotypes involving an anatomy term on anatomy pages?

**Comment by matentzn** [comment]
I understood this issue to be about rewiring MA related data that is imported straight to Uberon - what do you mean by "this" in "Will this effect"?
